core/src/test/scala/org/apache/spark/deploy/master/MasterSuite.scala

```diff
@@ -421,8 +421,7 @@ class MasterSuite extends SparkFunSuite
         // with LocalCluster, we have masters and workers in the same JVM, each overwriting
         // system property spark.ui.proxyBase.
         // so we need to manage this property explicitly for test
-        System.getProperty("spark.ui.proxyBase") should startWith
-          (s"$reverseProxyUrl/proxy/worker-")
+        System.getProperty("spark.ui.proxyBase") should startWith (reverseProxyUrl)
```

Comment:
Thanks for the fix!
Then we should remove line 421~423, and line 425 too

Reply:
ok ~

Reply:
513fc3e cleanup these lines and 0c77a48 change `startWith` to `be`.